integrations/github/jwt.go

```diff
@@ -112,7 +116,11 @@ func newClientFromGitHubAppID(appID int64) (*github.Client, error) {
 	return client, nil
 }
 
-func getPrivateKey() []byte {
+func getPrivateKey(privateKey string) []byte {
```

Comment:
I'm fine with merging this as-is because we'll probably refactor it soon enough, but it's better to call `getPrivateKey` as early as possible (in/before line line 57) rather than "deep" in line 97.

In other words, the general principle is that when you're passing a value (e.g. a private key), it's better to know that you're passing the right one from the start or as soon as possible, rather than passing something which is tentative and may change just before it's being used.

It's not an absolute principle, but obviously it promotes less cognitive load on the reader of the code.